transforms/code/repo_level_ordering/ray/README.md

```diff
@@ -79,81 +82,76 @@ To see results of the transform.
+This configuration can be used when data is on S3 like COS storage and computation is done
+on a ray cluster.
 
+Recommended `repo_lvl_store_type` for cluster is 'ray'. We need to dedicate some ray actors specially for
+ray store.
 
-local_conf="{'input_folder' : '$INPUT_FOLDER', 'output_folder' : '$OUTPUT_FOLDER'  }"
+for example, if we have a ray cluster with 100 actors available. We can dedicate some for the store. Let us randomly
+use 80 for computation and 10 for store and let 10 remain free.
+We need to add the following cli args:
 
-python src/repo_level_order_transform_ray.py \
-       --run_locally True \
-       --data_local_config "$local_conf" \
-       --repo_lvl_store_type  local  \
-       --repo_lvl_store_backend_dir '/tmp/mystore' \
-       --repo_lvl_combine_rows \
-       --repo_lvl_sorting_enabled \
-       --repo_lvl_sorting_algo SORT_SEMANTIC \
-       --repo_lvl_output_by_langs
-        
-```
+ `--runtime_num_workers 80` 
+ `--repo_lvl_store_type "ray" --repo_lvl_store_ray_cpus 0.2 --repo_lvl_store_ray_nworkers 10` 
 
-Case 2:
+When we want the output with the following configuration enabled:
 
-When data is on cloud and 
+- repos should be sorted by semantic order
+- output should be grouped by dominant language per repo
+- data is on s3
+- store backend : ray
 
-```shell
+> NOTE: store_backend=`s3/local` are persistent stores and retain the mappings stores in them, they need to be cleaned/deleted after use. It is recommented to use a different location for store if data is different. They are added to aid in large data processing in multiple stages.
+  store_backend=`ray`, is not presistent and ephimeral and does not retain any data. It is the simplest to use if resources are available.
+ 
 
+```sh
 
-export INPUT_FOLDER="input_bucket/input_dir"
-export OUTPUT_FOLDER="output_bucket/output_dir"
+export INPUT_FOLDER="input_cos_bucket"
+export OUTPUT_FOLDER="output_cos_bucket"
 
-s3_kreds="{ ... }" # in the deafult way used for all transforms. 
+s3_kreds="{ ... }" # in the default way used for all transforms. 
 s3_conf="{'input_folder' : '$INPUT_FOLDER', 'output_folder' : '$OUTPUT_FOLDER'  }"
 
-python src/repo_level_order_transform_ray.py \
-       --run_locally True \
-       --data_s3_cred "$s3_kreds" \
-       --data_s3_config "$s3_conf" \
-       --repo_lvl_store_type  local  \
-       --repo_lvl_store_backend_dir '/tmp/mystore' \
-       --repo_lvl_combine_rows \
-       --repo_lvl_sorting_enabled \
-       --repo_lvl_sorting_algo SORT_SEMANTIC \
-       --repo_lvl_output_by_langs   
+python repo_level_order_transform_ray.py  \
+             --data_s3_cred "$s3_kreds" \
+             --data_s3_config "$s3_conf" \
+             --runtime_num_workers 80 \
+             --repo_lvl_store_type "ray" \  
+             --repo_lvl_store_ray_cpus 0.2 \
+             --repo_lvl_store_ray_nworkers 10 \
+             --repo_lvl_sorting_enabled  \
+             --repo_lvl_output_by_langs 
+  
```

Comment:
Can you add the short section on running as an image. For example, https://github.com/IBM/data-prep-kit/blob/dev/transforms/universal/noop/ray/README.md#transforming-data-using-the-transform-image